rclcpp/include/rclcpp/parameter_value.hpp

```diff
@@ -25,6 +25,7 @@
 #include "rcl_interfaces/msg/parameter_type.hpp"
 #include "rcl_interfaces/msg/parameter_value.hpp"
 #include "rclcpp/visibility_control.hpp"
+#include "rclcpp/exceptions/exceptions.hpp"
```

Comment:
one last suggestion :grin: 

```suggestion
#include "rclcpp/exceptions/exceptions.hpp"
#include "rclcpp/visibility_control.hpp"
```